tests/integration/ceph_base_object_test.go

```diff
@@ -269,7 +268,7 @@ func testObjectStoreOperations(s suite.Suite, helper *clients.TestClient, k8sh *
 		logger.Info("OBC, Secret and ConfigMap created")
 	})
 
-	t.Run("use S3 client to put and get objects on OBC bucket", func(t *testing.T) {
+	t.Run("test S3 get/put/delete", func(t *testing.T) {
```

Comment:
Shorten this so the test name doesn't run past the edge of the screen in GH action run result.s